SlidingSegmentedControlTests/SlidingSegmentedControlTests.swift

```diff
@@ -82,6 +82,16 @@ class SlidingSegmentedControlTests: XCTestCase {
         XCTAssertEqual(slidingSegmentedControl.imageViews.map {$0.contentMode}, [.center, .center, .center])
     }
 
+    func testImageViewsIsAccessibilityElement() {
+        let slidingSegmentedControl = SlidingSegmentedControl(images: [UIImage(), UIImage()])
+        XCTAssertEqual(slidingSegmentedControl.imageViews.map { $0.isAccessibilityElement }, [true, true])
```

Comment:
Not a big fan of "clever" code in tests. Compare:

Complex:
```swift
let slidingSegmentedControl = SlidingSegmentedControl(images: [UIImage(), UIImage()])
XCTAssertEqual(slidingSegmentedControl.imageViews.map { $0.isAccessibilityElement }, [true, true])
```

Simple:
```swift
let image1 = UIImage()
let image2 = UIImage()

let slidingSegmentedControl = SlidingSegmentedControl(images: [image1, image2])

XCTAssertTrue(image1.isAccessibilityElement)
XCTAssertTrue(image2.isAccessibilityElement)
```

Now, if you were to maintain 1000 new unit tests, which style would you prefer? 😄 

P.S. An alternative to 2 asserts (not a fan as it is more complex again):
```swift
XCTAssertTrue(image1.isAccessibilityElement && image2.isAccessibilityElement)
```